{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -98,17 +99,9 @@ select = [
   "RET",
   "RUF",
   "SIM",
-  "W"
-]
-ignore = [
-  "ANN101",
-  "ANN401",
-  "E402",
-  "E501",
-  "F401",
-  "F841",
-  "RET504"
+  "W",
 ]
```

Comment:
Why the formatting changes to the TOML? :-)

Reply:
Probably an error